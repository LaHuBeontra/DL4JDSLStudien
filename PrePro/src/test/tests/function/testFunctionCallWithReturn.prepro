function main() {
    import vec3 p1, vec3 p2;

    nop();
    vec3 result = add(p1, p2);
    nop();

    export result;
}

function nop() {

}

function add(vec3 a, vec3 b) returns vec3 {
    return a + b;
}