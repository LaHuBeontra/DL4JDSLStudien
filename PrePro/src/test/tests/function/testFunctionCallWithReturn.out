DefaultPreProDataSet{
dataSetEntries=
[DataSetEntry{variableName='result', variableShape=[4, 3], startColumn=0, length=3}],
dataSet=
[[85.00,  0.00,  0.00],  
 [44,663.00,  0.00,  0.00],  
 [11.00,  0.00,  0.00],  
 [13.00,  0.00,  0.00]]

constants={}

calculated_variables=
result=[[85.00,  0.00,  0.00],  
 [44,663.00,  0.00,  0.00],  
 [11.00,  0.00,  0.00],  
 [13.00,  0.00,  0.00]]
}